src/adapter/graphql/tests/tests/test_gql_dataset_flow_runs.rs

```diff
@@ -690,6 +825,16 @@ impl FlowRunsHarness {
             .add_value(FlowServiceRunConfig::new(chrono::Duration::seconds(1)))
             .add::<TaskSchedulerInMemory>()
             .add::<TaskSystemEventStoreInMemory>()
+            .add_builder(
+                PollingIngestServiceImpl::builder()
+                    .with_cache_dir(cache_dir)
+                    .with_run_info_dir(run_info_dir)
+                    .with_container_runtime(Arc::new(ContainerRuntime::default()))
```

Comment:
Why not `.add::<ContainerRuntime>` instead of constructing it here?

Reply:
Simplified